bundles/net.tourbook.device/src/net/tourbook/device/polar/hrm/messages_fr.properties

```diff
@@ -1,28 +1,71 @@
 #Generated by Eclipse Messages Editor (Eclipse Babel)
 
-Import_Confirm_PPD = 
+Import_Confirm_PPD = Vous \u00EAtes sur le point d'importer un fichier PPD (Polar Person Definition).\n\
```

Comment:
@wolfgang-ch I recently found out that the Polar Preference page didn't even have the french translation file.

Reply:
Translation files are not automatically created with native Eclipse, one of the messages editors may can create the language files as I remember.

Reply:
What I meant is that I am surprised nobody noticed/complained that the complete Polar preference page french translation was missing all those years (5+ years ago!)